src/foam/core/Listener.js

```diff
@@ -87,10 +103,11 @@ foam.CLASS({
       var isMerged   = this.isMerged;
       var isFramed   = this.isFramed;
       var mergeDelay = this.mergeDelay;
+      var on         = this.on;
 
-      Object.defineProperty(proto, name, {
+      var obj = Object.defineProperty(proto, name, {
```

Comment:
Why store in 'obj'? Isn't that just the same object as 'proto'?

Reply:
Yes